### PR TITLE
Update version and changelog for 1.1 release

### PR DESCRIPTION
**Pull request by prashantv**

### CODE REVIEW

**Comment by yurishkuro** [comment]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+  As long as a Zipkin-style tracing is configured, TChannel frames still
+  send tracing information, and `CurrentSpan(ctx)` works as above.
+  All tracer configuration must be handled through OpenTracing.
+  (#426)
+* Improve error messages when using the json package and the host:port
+  fails to connect. (475)
+* mockhyperbahn now using inbuilt TChannel relaying to implement in-process
+  forwarding. (#472)
+* Drop go1.4 support and add support for go1.7.
+* Drop support for go1.4, update 1.7 to rc3 (#471)
```
Comment (akshayjshah) [inline CHANGELOG.md]: These are duplicates; probably only need one in the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+# v1.1.0
+
+* Integrate OpenTracing for distributed tracing and context propagation.
+  As long as a Zipkin-style tracing is configured, TChannel frames still
+  send tracing information, and `CurrentSpan(ctx)` works as above.
```
Comment (akshayjshah) [inline CHANGELOG.md]: What does "as above" mean in this context?

Comment (prashantv) [inline CHANGELOG.md]: fixed, meant to be "as before".

File: CHANGELOG.md (hunk 1 above)
Comment (prashantv) [inline CHANGELOG.md]: this is the original PR, meant to delete this after writing the above. Done

**Pull request merged**